### PR TITLE
Update the regular expression to work "v" prefix in the version name

### PR DESCRIPTION
**Pull request by Gustry**
Well done for the regexp @Guts ;-)

I could fix the "v" prefix very easily.

But writing the test for all versions in the changelog file showed us that a sub section doesn't work, version `10.1.0-alpha1`.
Should we go ahead and merge anyway ?

### CODE REVIEW

**Comment by 3nids** [comment]
May I suggest to update the regex101 page with new regex and examples?

**Comment by Gustry** [comment]
Of course you may. Fixed indeed.

**Comment by 3nids** [comment]
is this OK to merge?
it seems 10.1.0-alpha1 is correctly found by the regex no?

File: test/test_changelog.py [hunk 1]
```diff
@@ -54,16 +54,37 @@ def test_changelog_content(self):
...
+                "- This is the latest documented version in this changelog\n"
+                "- The changelog module is tested against these lines\n"
+                "- Be careful modifying this file"
+            ),
+            "10.1.0-alpha1": (
+                "- This is a version with a prerelease in this changelog\n"
+                "- The changelog module is tested against these lines\n"
+                "- Be careful modifying this file"
+                # "\n" TODO Fixed section is missing
+                # "- trying with a subsection in a version note"
```
Comment (Gustry) [inline test/test_changelog.py:74]: This is not working with the current regex it seems.
It's in the changelog : https://github.com/opengisch/qgis-plugin-ci/blob/master/test/fixtures/CHANGELOG.md#1010-alpha1---20210110 using a sub section

```
## Added
## Changed
## Removed
```

**Comment by Gustry** [comment]
> is this OK to merge?

For me yes

I think the bug I have noticed here can be fixed later, we are not using it for now.

So it's still not fully compatible with the format "Keep a Changelog"

File: test/test_changelog.py (hunk 1 above)
Comment (Guts) [reply test/test_changelog.py:74]: I think we should priorize the compatibility with the conventional Keep a Changelog over specific usage, no?

Comment (Guts) [reply test/test_changelog.py:74]: Ok, sorry I did not see that's already merged. Too late! My bad :wink: .

**Pull request merged**